### ISSUE TITLE
When to play animations

### ISSUE THREAD

**Issue by BreeadHead** [microsoft/minecraft-samples#11]
I don't really know if this is the right place to say this, but I'm having trouble figuring out how to make a custom entity I made play animations at the right time. 
I have an entity that uses the behaviors random_stroll and nearest_attackable_target, and I made animations in blockbench called "wander" and "chase". I'm trying to figure out how to make it play the wander animation when it's wandering around and stops when the entity is still, and how to play the chase animation when it finds a target.

**Comment by jacko-png** [comment]
No, this isn't the best place to say this. If you haven't already, try in r/BedrockAddons.
